packages/portalnetwork/src/client/client.ts

```diff
@@ -391,6 +391,11 @@ export class PortalNetwork extends EventEmitter<PortalNetworkEvents> {
   ) => {
     this.metrics?.totalBytesReceived.inc(message.request.length)
     if (bytesToHex(message.protocol) === NetworkId.UTPNetwork) {
+      if (!this.discv5.findEnr(nodeAddress.nodeId)) {
```

Comment:
This doesn't seem like it should work correctly.  Why are we using the discv5 routing table to lookup nodes that are very likely not in the discv5 routing table but would be in one of the network routing tables (assuming it's a real node that we are interacting with and likely have a uTP socket with)?

Reply:
Ah, yeah you're right.  checking the routing table doesn't necessarily tell us whether we know this peer or not.

Reply:
Deleted this.  It was overkill anyway, as the utp handler will check if for any history with that peer